### ISSUE TITLE
MA0076 warns about using implicit culture-sensitive ToString for TimeSpan even though its invariant by default

### ISSUE THREAD

**Issue by Piedone** [meziantou/Meziantou.Analyzer#545]
### Version of the Meziantou.Analyzer NuGet package

2.0.58

### Rule Identifier

MA0076

### Target Framework

net6.0

### C# Language version

C# 11

### Description

[MA0076](https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0076.md) produces a violation for `TimeSpan.Tostring()` even though that uses a culture-insensitive string representation, see [the docs](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings#the-constant-c-format-specifier).

### Reproduction Steps

Minimal code:

```c#
var timeSpan = TimeSpan.FromSeconds(1);
var myString = $"This is a test: {timeSpan}";
```

Using `$"This is a test: {timeSpan:c}"` doesn't produce a violation, correctly, but the above code is in effect the same.
